src/components/projects/CreateProjectForm.vue

```diff
@@ -189,6 +190,20 @@
         >
           <b-checkbox v-model="longstackIsActive" />
         </b-field>
+
+        <b-field
+          style="margin-left: 2em;"
+          label="Deplete"
+        >
+          <b-checkbox v-model="depleteIsActive" />
+        </b-field>
+
+        <b-field
+          style="margin-left: 2em;"
```

Comment:
In general it's preferable to keep literal css out of the html and use classes instead. In this case I see you're just following convention above, and it's a quick addition, so I'm not saying change it here. Just worth thinking about, in the same spirit as our recent linting improvements.